src/main/java/datawave/query/util/AllFieldMetadataHelper.java

```diff
@@ -142,37 +156,62 @@ protected String getCompositeFieldName(Key k) {
         return compositeFieldName;
     }
     
+    /**
+     * Get the authorizations used by this helper
+     * 
+     * @return the authorizations
+     */
     public Set<Authorizations> getAuths() {
         return auths;
     }
     
+    /**
+     * Get the full user authorizations used by this helper
+     * 
+     * @return the full user authorizations
+     */
     public Set<Authorizations> getFullUserAuths() {
         return fullUserAuths;
     }
     
+    /**
+     * Get the metadata table name
+     * 
+     * @return the metadata table name
+     */
     public String getMetadataTableName() {
         return metadataTableName;
     }
     
+    /**
+     * Get the {@link TypeMetadataHelper}
+     * 
+     * @return the TypeMetadataHelper
+     */
     public TypeMetadataHelper getTypeMetadataHelper() {
         return typeMetadataHelper;
     }
     
     /**
-     * Method that determines whether or not a column exists in the metadata table for the given key.
+     * Method that determines whether a column exists in the metadata table for the given key.
      * 
      * @param colf
+     *            the column family
      * @param key
-     * @return
+     *            the key, an Entry of table name, field, and ingest types
+     * @return true if the key exists in the metadata table
      * @throws TableNotFoundException
+     *             if no table exists
      * @throws InstantiationException
+     *             not thrown, remove
```

Comment:
this is more for my curiosity; is the removal of these superfluous exceptions intended to be done in another ticket?

Reply:
Yes, cleaning up the thrown exceptions is intended to be done later. It's technically breaking the method contract, so updates need to be made to any project that uses the metadata helper. That set of changes was extensive enough that I wanted to do it separately.